frontend/src/store/index.ts

```diff
@@ -219,5 +224,33 @@ export const store = createStore<State>({
         commit('setHobbit', json)
       })
     },
+    async createWebSocketConnection({ commit, dispatch }) {
+      const socket = new WebSocket('ws://localhost:8080/ws')
+      console.log(socket)
+      socket.onmessage = (ev) => {
+        console.log(ev)
+        const parsedEventData = JSON.parse(ev.data)
```

Comment:
we might want to take a look what's also part of ev and if we can somehow use it